2024-10-01 12:05:31,740 [ERROR] root: Error processing dice: Invalid dice expression: 3d
2024-10-01 12:05:35,990 [ERROR] root: Error processing dice: Invalid dice expression: d
2024-10-01 12:05:39,415 [WARNING] root: DB connection closed
2024-10-01 12:05:39,425 [WARNING] root: DB connection closed
2024-10-01 12:06:22,931 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "stats" is not found
2024-10-01 12:06:42,022 [WARNING] root: DB connection closed
2024-10-14 11:22:28,490 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 11:22:28,491 [INFO] discord.client: logging in using static token
2024-10-14 11:22:29,441 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 17273bfc38a86cbdd315352a69db6f4b).
2024-10-14 11:22:33,151 [ERROR] root: Error processing dice: invalid literal for int() with base 10: ''
2024-10-14 11:37:47,736 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 11:37:47,737 [INFO] discord.client: logging in using static token
2024-10-14 11:37:48,634 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: d623599236dd70654d6332d948753f62).
2024-10-14 11:42:58,146 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 11:42:58,147 [INFO] discord.client: logging in using static token
2024-10-14 11:42:59,491 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 73fd81684fb4cfad81126d1e768cb845).
2024-10-14 11:44:35,438 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 11:44:35,440 [INFO] discord.client: logging in using static token
2024-10-14 11:44:36,389 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 3b9c54e92bfadb28f078cd07aa5d7e67).
2024-10-14 11:45:05,780 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 11:45:05,781 [INFO] discord.client: logging in using static token
2024-10-14 11:45:07,765 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 65b44715952c7c5508aeea4db8e090a5).
2024-10-14 11:47:29,821 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 11:47:29,822 [INFO] discord.client: logging in using static token
2024-10-14 11:47:30,755 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 01a531c19caa3b8db40cd25224846d84).
2024-10-14 11:47:48,883 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 11:47:48,885 [INFO] discord.client: logging in using static token
2024-10-14 11:47:50,241 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: cc478392522a01a320ff476f84a7b7fe).
2024-10-14 11:47:50,914 [ERROR] root: Error rolling initiative: 'list' object has no attribute '__dict__'
2024-10-14 12:01:25,770 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 12:01:25,771 [INFO] discord.client: logging in using static token
2024-10-14 12:01:26,702 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: cc7a6678b3b3fb367e4e3edbcbc573c7).
2024-10-14 12:17:42,856 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 12:17:42,858 [INFO] discord.client: logging in using static token
2024-10-14 12:17:43,928 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 846e08c3e101220041117d3acbbc7046).
2024-10-14 15:15:18,391 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 15:31:07,439 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 15:37:46,418 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 15:43:51,011 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 15:46:42,298 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 15:46:42,299 [INFO] discord.client: logging in using static token
2024-10-14 15:46:43,395 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1900f32d86951e322d06ead8c59ac873).
2024-10-14 15:46:54,164 [ERROR] root: Error processing dice: can only join an iterable
2024-10-14 15:47:28,171 [ERROR] root: Error processing dice: can only join an iterable
2024-10-14 15:48:29,012 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 15:48:29,013 [INFO] discord.client: logging in using static token
2024-10-14 15:48:29,964 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: ebd963743eb935823bbcc8503f9e6b99).
2024-10-14 15:48:31,876 [ERROR] core.dice_engine: Invalid dice expression: !r
2024-10-14 15:48:31,876 [ERROR] root: Error processing dice: Invalid dice expression: !r
2024-10-14 15:48:35,239 [ERROR] core.dice_engine: Invalid dice expression: !
2024-10-14 15:48:35,239 [ERROR] root: Error processing dice: Invalid dice expression: !
2024-10-14 15:48:39,167 [ERROR] root: Error processing dice: Invalid remaining data in expression: +!r
2024-10-14 17:48:02,204 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 18:05:06,984 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 18:05:16,443 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 18:05:16,445 [INFO] discord.client: logging in using static token
2024-10-14 18:05:17,486 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 81eda4d33f89866dba3d944ad6bbf539).
2024-10-14 18:05:29,830 [ERROR] discord.app_commands.tree: Ignoring exception in command 'r'
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/app_commands/tree.py", line 1310, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/hybrid.py", line 442, in _invoke_with_namespace
    await command.prepare(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 938, in prepare
    await self._parse_arguments(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/hybrid.py", line 539, in _parse_arguments
    ctx.kwargs = await self.app_command._transform_arguments(interaction, interaction.namespace)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/hybrid.py", line 357, in _transform_arguments
    raise app_commands.CommandSignatureMismatch(self) from None
discord.app_commands.errors.CommandSignatureMismatch: The signature for command 'r' is different from the one provided by Discord. This can happen because either your code is out of date or you have not synced the commands with Discord, causing the mismatch in data. It is recommended to sync the command tree to fix this issue.
2024-10-14 18:05:35,310 [ERROR] discord.app_commands.tree: Ignoring exception in command 'r'
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/app_commands/tree.py", line 1310, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/hybrid.py", line 442, in _invoke_with_namespace
    await command.prepare(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 938, in prepare
    await self._parse_arguments(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/hybrid.py", line 539, in _parse_arguments
    ctx.kwargs = await self.app_command._transform_arguments(interaction, interaction.namespace)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/hybrid.py", line 357, in _transform_arguments
    raise app_commands.CommandSignatureMismatch(self) from None
discord.app_commands.errors.CommandSignatureMismatch: The signature for command 'r' is different from the one provided by Discord. This can happen because either your code is out of date or you have not synced the commands with Discord, causing the mismatch in data. It is recommended to sync the command tree to fix this issue.
2024-10-14 18:10:17,240 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 18:10:17,242 [INFO] discord.client: logging in using static token
2024-10-14 18:10:18,167 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 9ff6efc15f6752b093b85acbb7596d3f).
2024-10-14 18:10:47,931 [INFO] root: Command tree synced.
2024-10-14 18:12:21,723 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 18:12:21,725 [INFO] discord.client: logging in using static token
2024-10-14 18:12:22,804 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: ef7062d61f9cc0f28b367983f253e22f).
2024-10-14 18:14:20,097 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 18:14:29,633 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 18:14:29,634 [INFO] discord.client: logging in using static token
2024-10-14 18:14:30,867 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: af775d87af799159ec124c53c03641a8).
2024-10-14 18:17:02,644 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 18:17:02,646 [INFO] discord.client: logging in using static token
2024-10-14 18:17:03,583 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 82b33616075eb4698dd2137ce735cd48).
2024-10-14 18:17:07,085 [ERROR] discord.ext.commands.bot: Ignoring exception in command sync
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/commands/debug.py", line 26, in sync
    await bot.tree.sync()
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/app_commands/tree.py", line 1134, in sync
    raise CommandSyncFailure(e, commands) from None
discord.app_commands.errors.CommandSyncFailure: Failed to upload commands to Discord (HTTP status 400, error code 50035)
In command 'critic' defined in function 'commands_dice.<locals>.command_roll_critical_damage_dice'
  description: Must be 100 or fewer in length.
In command 'd' defined in function 'commands_dice.<locals>.command_roll_disadvantage_dice'
  description: Must be 100 or fewer in length.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: CommandSyncFailure: Failed to upload commands to Discord (HTTP status 400, error code 50035)
In command 'critic' defined in function 'commands_dice.<locals>.command_roll_critical_damage_dice'
  description: Must be 100 or fewer in length.
In command 'd' defined in function 'commands_dice.<locals>.command_roll_disadvantage_dice'
  description: Must be 100 or fewer in length.
2024-10-14 18:23:49,507 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-14 18:23:49,509 [INFO] discord.client: logging in using static token
2024-10-14 18:23:50,582 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 6c7a4d4e5ef5b52209196df9ca1acb7d).
2024-10-14 18:23:54,340 [INFO] root: Command tree synced.
2024-10-15 00:03:48,538 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:05:25,933 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:05:31,202 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:06:14,066 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:06:51,893 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:08:35,032 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:08:54,959 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:09:28,514 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:15:49,214 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:15:49,216 [INFO] discord.client: logging in using static token
2024-10-15 00:15:50,166 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 316ab376bfe05809ab6d8e8f2d4bb23d).
2024-10-15 00:19:09,583 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:19:09,585 [INFO] discord.client: logging in using static token
2024-10-15 00:19:10,521 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: c50285dc80fa80ee761e8deefae3af56).
2024-10-15 00:19:22,007 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "hello_world" is not found
2024-10-15 00:20:38,367 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:20:38,369 [INFO] discord.client: logging in using static token
2024-10-15 00:20:39,727 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 611dc9b190be8774bc78acefad1362ed).
2024-10-15 00:21:15,049 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:21:15,051 [INFO] discord.client: logging in using static token
2024-10-15 00:21:15,907 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 31ce7fc0d1888e19bbcb831839f6fb1f).
2024-10-15 00:21:36,035 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:21:36,037 [INFO] discord.client: logging in using static token
2024-10-15 00:21:37,085 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 996c4f9d2a6199cb55929dc21fe71180).
2024-10-15 00:22:11,945 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:22:11,946 [INFO] discord.client: logging in using static token
2024-10-15 00:22:12,882 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: f911212917711e85d302d3ed1f6bc1be).
2024-10-15 00:24:14,604 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:24:14,606 [INFO] discord.client: logging in using static token
2024-10-15 00:24:15,490 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 35e34917faa8b37e3a7eb788211fffa0).
2024-10-15 00:24:34,187 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:24:34,189 [ERROR] config: Config file .config.json not found.
2024-10-15 00:24:47,643 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:24:47,645 [ERROR] config: Config file config.json not found.
2024-10-15 00:25:50,786 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 00:25:50,788 [INFO] discord.client: logging in using static token
2024-10-15 00:25:51,641 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 5ae970ad274229abe00d59a2113d789f).
2024-10-15 00:25:58,802 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "hellow_world" is not found
2024-10-15 11:07:45,795 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 11:07:45,798 [INFO] discord.client: logging in using static token
2024-10-15 11:07:46,824 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 68425c26b1a9a82983046f9eab38fc2f).
2024-10-15 12:39:48,943 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:39:48,944 [INFO] discord.client: logging in using static token
2024-10-15 12:39:49,964 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 7f3180344b92ee4b0f38c0250330c3a3).
2024-10-15 12:40:10,694 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "magic" is not found
2024-10-15 12:40:53,657 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:40:53,659 [INFO] discord.client: logging in using static token
2024-10-15 12:40:54,644 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: c1425682cd676b8a2440b5c6fab9c81d).
2024-10-15 12:41:13,536 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:41:13,537 [INFO] commands: Loaded 0 plugins
2024-10-15 12:41:13,537 [INFO] discord.client: logging in using static token
2024-10-15 12:41:14,813 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: a96f60246892f4aff28eed655d8f4db1).
2024-10-15 12:42:10,380 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:42:10,381 [INFO] commands: Loading 2 plugins
2024-10-15 12:42:10,382 [INFO] root: Plugin PluginMagic initialized!
2024-10-15 12:42:10,382 [INFO] commands: Plugin: type ready!
2024-10-15 12:42:10,382 [INFO] root: Plugin PluginExample initialized!
2024-10-15 12:42:10,382 [INFO] commands: Plugin: type ready!
2024-10-15 12:42:10,382 [INFO] discord.client: logging in using static token
2024-10-15 12:42:11,322 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 14752b12d5b95bc8c6bf90ad30473c1a).
2024-10-15 12:43:32,285 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:43:32,287 [INFO] commands: Loading 2 plugins
2024-10-15 12:43:32,287 [INFO] root: PluginMagic initialized!
2024-10-15 12:43:32,287 [INFO] commands: Plugin: type ready!
2024-10-15 12:43:32,287 [INFO] root: PluginExample initialized!
2024-10-15 12:43:32,288 [INFO] commands: Plugin: type ready!
2024-10-15 12:43:32,288 [INFO] discord.client: logging in using static token
2024-10-15 12:43:33,320 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: a2d708db1cb5e062f23c73a4b7c860c5).
2024-10-15 12:44:26,290 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:44:26,292 [INFO] commands: Loading 2 plugins
2024-10-15 12:44:26,292 [INFO] root: PluginMagic initialized!
2024-10-15 12:44:26,292 [INFO] commands: Plugin: type ready!
2024-10-15 12:44:26,292 [INFO] root: PluginExample initialized!
2024-10-15 12:44:26,292 [INFO] commands: Plugin: type ready!
2024-10-15 12:44:26,292 [INFO] discord.client: logging in using static token
2024-10-15 12:44:27,492 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1faf93cb6a9d1e8f69ca204037b67cbf).
2024-10-15 12:44:56,517 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:44:56,519 [INFO] commands: Loading 2 plugins
2024-10-15 12:44:56,519 [INFO] root: PluginMagic initialized!
2024-10-15 12:44:56,519 [INFO] commands: Plugin: type ready!
2024-10-15 12:44:56,519 [INFO] root: PluginExample initialized!
2024-10-15 12:44:56,519 [INFO] commands: Plugin: type ready!
2024-10-15 12:44:56,519 [INFO] discord.client: logging in using static token
2024-10-15 12:44:57,501 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: eaeebd2314e13dc34ede4759e51a8a5d).
2024-10-15 12:45:13,738 [ERROR] discord.ext.commands.bot: Ignoring exception in command magic
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 29, in say_hello_world
    data = json.load(f)
  File "/usr/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.10/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 40 column 1 (char 1088)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: JSONDecodeError: Expecting property name enclosed in double quotes: line 40 column 1 (char 1088)
2024-10-15 12:48:36,402 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:48:36,404 [INFO] commands: Loading 2 plugins
2024-10-15 12:48:36,404 [INFO] root: PluginMagic initialized!
2024-10-15 12:48:36,404 [INFO] commands: Plugin: type ready!
2024-10-15 12:48:36,404 [INFO] root: PluginExample initialized!
2024-10-15 12:48:36,404 [INFO] commands: Plugin: type ready!
2024-10-15 12:48:36,405 [INFO] discord.client: logging in using static token
2024-10-15 12:48:37,497 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 2100fbf78eba30a828d7dcef817c6ec6).
2024-10-15 12:48:39,254 [ERROR] discord.ext.commands.bot: Ignoring exception in command magic
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 29, in say_hello_world
    data = json.load(f)
  File "/usr/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.10/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 40 column 1 (char 1088)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: JSONDecodeError: Expecting property name enclosed in double quotes: line 40 column 1 (char 1088)
2024-10-15 12:50:04,890 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:50:04,892 [INFO] commands: Loading 2 plugins
2024-10-15 12:50:31,609 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:50:31,611 [INFO] commands: Loading 2 plugins
2024-10-15 12:50:56,473 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:50:56,475 [INFO] commands: Loading 2 plugins
2024-10-15 12:50:56,475 [INFO] root: PluginMagic initialized!
2024-10-15 12:50:56,475 [INFO] commands: Plugin: type ready!
2024-10-15 12:50:56,475 [INFO] root: PluginExample initialized!
2024-10-15 12:50:56,475 [INFO] commands: Plugin: type ready!
2024-10-15 12:50:56,475 [INFO] discord.client: logging in using static token
2024-10-15 12:50:57,567 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 2f5d77ab802d006923ed9e9f84aa7632).
2024-10-15 12:55:56,011 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:55:56,013 [INFO] commands: Loading 2 plugins
2024-10-15 12:55:56,013 [INFO] root: PluginMagic initialized!
2024-10-15 12:55:56,013 [INFO] commands: Plugin: type ready!
2024-10-15 12:55:56,013 [INFO] root: PluginExample initialized!
2024-10-15 12:55:56,013 [INFO] commands: Plugin: type ready!
2024-10-15 12:55:56,014 [INFO] discord.client: logging in using static token
2024-10-15 12:55:56,892 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 83d3ede886dd06ad6cccd09f42aa12ac).
2024-10-15 12:56:00,264 [ERROR] discord.ext.commands.bot: Ignoring exception in command magic
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 49, in say_hello_world
    descriptions = find_effect(roll.total_dice_result)
TypeError: find_effect() missing 1 required positional argument: 'data'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: find_effect() missing 1 required positional argument: 'data'
2024-10-15 12:56:23,637 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:56:23,639 [INFO] commands: Loading 2 plugins
2024-10-15 12:56:23,639 [INFO] root: PluginMagic initialized!
2024-10-15 12:56:23,639 [INFO] commands: Plugin: type ready!
2024-10-15 12:56:23,639 [INFO] root: PluginExample initialized!
2024-10-15 12:56:23,639 [INFO] commands: Plugin: type ready!
2024-10-15 12:56:23,640 [INFO] discord.client: logging in using static token
2024-10-15 12:56:24,592 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: e92a5b3bbe08f79b53f101184339dbb8).
2024-10-15 12:56:27,002 [ERROR] discord.ext.commands.bot: Ignoring exception in command magic
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 52, in say_hello_world
    descriptions = find_effect(roll.total_dice_result, effects)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 22, in find_effect
    for effect in data['effects']:
TypeError: list indices must be integers or slices, not str

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: list indices must be integers or slices, not str
2024-10-15 12:57:21,377 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:57:21,379 [INFO] commands: Loading 2 plugins
2024-10-15 12:57:21,379 [INFO] root: PluginMagic initialized!
2024-10-15 12:57:21,379 [INFO] commands: Plugin: type ready!
2024-10-15 12:57:21,380 [INFO] root: PluginExample initialized!
2024-10-15 12:57:21,380 [INFO] commands: Plugin: type ready!
2024-10-15 12:57:21,380 [INFO] discord.client: logging in using static token
2024-10-15 12:57:22,337 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 12adfa951bfc0eb5f3136226c9a4c0b3).
2024-10-15 12:57:23,843 [ERROR] discord.ext.commands.bot: Ignoring exception in command magic
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 52, in say_hello_world
    descriptions = find_effect(roll.total_dice_result, effects)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 23, in find_effect
    if effect['range'] <= roll <= effect['range']:
TypeError: '<=' not supported between instances of 'list' and 'int'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: '<=' not supported between instances of 'list' and 'int'
2024-10-15 12:57:51,943 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:57:51,945 [INFO] commands: Loading 2 plugins
2024-10-15 12:57:51,946 [INFO] root: PluginMagic initialized!
2024-10-15 12:57:51,946 [INFO] commands: Plugin: type ready!
2024-10-15 12:57:51,946 [INFO] root: PluginExample initialized!
2024-10-15 12:57:51,946 [INFO] commands: Plugin: type ready!
2024-10-15 12:57:51,946 [INFO] discord.client: logging in using static token
2024-10-15 12:57:52,939 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 13c8bae64beb42832b418a68a526dba3).
2024-10-15 12:58:15,385 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-15 12:58:15,388 [INFO] commands: Loading 2 plugins
2024-10-15 12:58:15,388 [INFO] root: PluginMagic initialized!
2024-10-15 12:58:15,388 [INFO] commands: Plugin: type ready!
2024-10-15 12:58:15,388 [INFO] root: PluginExample initialized!
2024-10-15 12:58:15,388 [INFO] commands: Plugin: type ready!
2024-10-15 12:58:15,389 [INFO] discord.client: logging in using static token
2024-10-15 12:58:16,377 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 5ee49e5f4d575c37db93efc571d7f38e).
2024-10-15 14:09:40,361 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "maigc" is not found
2024-10-15 14:12:10,786 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 5ee49e5f4d575c37db93efc571d7f38e.
2024-10-15 14:39:35,631 [ERROR] discord.client: Attempting a reconnect in 1.52s
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/client.py", line 667, in connect
    await self.ws.poll_event()
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.10/lib/python3.10/site-packages/discord/gateway.py", line 648, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
2024-10-15 14:39:37,500 [INFO] discord.gateway: Shard ID None session has been invalidated.
2024-10-15 14:39:43,028 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: f06ab2c3f3b042a0153f6c4f53be7f38).
2024-10-17 13:04:00,032 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-17 13:04:00,033 [ERROR] config: KeyError: 'roll_dice'
2024-10-17 13:04:13,981 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-17 13:04:13,986 [INFO] commands: Loading 2 plugins
2024-10-17 13:04:13,987 [INFO] root: PluginMagic initialized!
2024-10-17 13:04:13,987 [INFO] commands: Plugin: type ready!
2024-10-17 13:04:13,987 [INFO] root: PluginExample initialized!
2024-10-17 13:04:13,987 [INFO] commands: Plugin: type ready!
2024-10-17 13:04:13,987 [INFO] discord.client: logging in using static token
2024-10-17 13:04:14,984 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 86604e554b42f27d6f56cf60025cdca0).
2024-10-17 13:50:03,571 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 86604e554b42f27d6f56cf60025cdca0.
2024-10-17 14:14:14,354 [ERROR] discord.ext.commands.bot: Ignoring exception in command magic
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 81, in say_hello_world
    effect = get_failed_magic_surge_effect(roll.total_dice_result,
                                    self.magic_fail_list)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 44, in get_failed_magic_surge_effect
    for effect in data[magic_surge_type]:
                  ~~~~^^^^^^^^^^^^^^^^^^
TypeError: list indices must be integers or slices, not str

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: list indices must be integers or slices, not str
2024-10-17 14:19:34,973 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-17 14:19:34,978 [INFO] commands: Loading 2 plugins
2024-10-17 14:19:34,979 [INFO] root: PluginMagic initialized!
2024-10-17 14:19:34,979 [INFO] commands: Plugin: type ready!
2024-10-17 14:19:34,979 [INFO] root: PluginExample initialized!
2024-10-17 14:19:34,979 [INFO] commands: Plugin: type ready!
2024-10-17 14:19:34,980 [INFO] discord.client: logging in using static token
2024-10-17 14:19:36,082 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 6bb5d3816dc398693124854a631eb4a8).
2024-10-17 14:20:59,028 [ERROR] discord.ext.commands.bot: Ignoring exception in command magic
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 87, in say_hello_world
    effect = get_failed_magic_surge_effect(roll.total_dice_result,
                                    self.magic_fail_list)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/magic/plugin_magic.py", line 44, in get_failed_magic_surge_effect
    for effect in data[magic_surge_type]:
                  ~~~~^^^^^^^^^^^^^^^^^^
TypeError: list indices must be integers or slices, not str

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: list indices must be integers or slices, not str
2024-10-17 14:22:11,889 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-17 14:22:11,892 [INFO] commands: Loading 2 plugins
2024-10-17 14:22:11,893 [INFO] root: PluginMagic initialized!
2024-10-17 14:22:11,893 [INFO] commands: Plugin: type ready!
2024-10-17 14:22:11,893 [INFO] root: PluginExample initialized!
2024-10-17 14:22:11,893 [INFO] commands: Plugin: type ready!
2024-10-17 14:22:11,893 [INFO] discord.client: logging in using static token
2024-10-17 14:22:12,828 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: a716629fe2b23e2265c4bf8c16ab4c5e).
2024-10-17 14:23:18,239 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-17 14:23:18,242 [INFO] commands: Loading 2 plugins
2024-10-17 14:23:18,243 [INFO] root: PluginMagic initialized!
2024-10-17 14:23:18,243 [INFO] commands: Plugin: type ready!
2024-10-17 14:23:18,243 [INFO] root: PluginExample initialized!
2024-10-17 14:23:18,243 [INFO] commands: Plugin: type ready!
2024-10-17 14:23:18,243 [INFO] discord.client: logging in using static token
2024-10-17 14:23:19,337 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 90973fcf1e99b7f3cddae837ff1a1edc).
2024-10-17 14:24:44,677 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-17 14:24:44,680 [INFO] commands: Loading 2 plugins
2024-10-17 14:24:44,680 [INFO] root: PluginMagic initialized!
2024-10-17 14:24:44,681 [INFO] commands: Plugin: type ready!
2024-10-17 14:24:44,681 [INFO] root: PluginExample initialized!
2024-10-17 14:24:44,681 [INFO] commands: Plugin: type ready!
2024-10-17 14:24:44,681 [INFO] discord.client: logging in using static token
2024-10-17 14:24:45,594 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: af68af31e2e4a948d6eada0521c30f8a).
2024-10-17 14:26:00,717 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-17 14:26:00,721 [INFO] commands: Loading 2 plugins
2024-10-17 14:26:00,721 [INFO] root: PluginMagic initialized!
2024-10-17 14:26:00,721 [INFO] commands: Plugin: type ready!
2024-10-17 14:26:00,722 [INFO] root: PluginExample initialized!
2024-10-17 14:26:00,722 [INFO] commands: Plugin: type ready!
2024-10-17 14:26:00,722 [INFO] discord.client: logging in using static token
2024-10-17 14:26:01,665 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 6fef4d7cf68625983b1fedfece48ac4b).
2024-10-19 13:10:16,981 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-19 13:10:16,986 [INFO] commands: Loading 2 plugins
2024-10-19 13:10:16,987 [INFO] root: PluginMagic initialized!
2024-10-19 13:10:16,987 [INFO] commands: Plugin: type ready!
2024-10-19 13:10:16,987 [INFO] root: PluginExample initialized!
2024-10-19 13:10:16,987 [INFO] commands: Plugin: type ready!
2024-10-19 13:10:16,988 [INFO] discord.client: logging in using static token
2024-10-19 13:10:18,007 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: f75f9faa84560118b3410fc149412ee0).
2024-10-19 13:10:35,305 [ERROR] core.dice_engine: Invalid dice expression: 0
2024-10-19 13:10:35,305 [ERROR] root: Error processing dice: Invalid dice expression: 0
2024-10-19 13:10:54,225 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-19 13:10:54,227 [INFO] commands: Loading 2 plugins
2024-10-19 13:10:54,228 [INFO] root: PluginMagic initialized!
2024-10-19 13:10:54,228 [INFO] commands: Plugin: type ready!
2024-10-19 13:10:54,228 [INFO] root: PluginExample initialized!
2024-10-19 13:10:54,228 [INFO] commands: Plugin: type ready!
2024-10-19 13:10:54,229 [INFO] discord.client: logging in using static token
2024-10-19 13:10:55,393 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: d736a00d821d6009d8076bfc7d5f2dcd).
2024-10-19 13:10:56,394 [ERROR] core.dice_engine: Invalid dice expression: 0
2024-10-19 13:10:56,395 [ERROR] root: Error processing dice: Invalid dice expression: 0
2024-10-19 13:11:07,957 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-19 13:11:07,959 [INFO] commands: Loading 2 plugins
2024-10-19 13:11:07,960 [INFO] root: PluginMagic initialized!
2024-10-19 13:11:07,960 [INFO] commands: Plugin: type ready!
2024-10-19 13:11:07,961 [INFO] root: PluginExample initialized!
2024-10-19 13:11:07,961 [INFO] commands: Plugin: type ready!
2024-10-19 13:11:07,961 [INFO] discord.client: logging in using static token
2024-10-19 13:11:08,861 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 0079ba574136121e28ccdf41163521c0).
2024-10-19 13:11:10,205 [ERROR] core.dice_engine: Invalid dice expression: 0
2024-10-19 13:11:10,205 [ERROR] root: Error processing dice: Invalid dice expression: 0
2024-10-19 13:11:25,294 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-19 13:11:25,296 [INFO] commands: Loading 2 plugins
2024-10-19 13:11:25,297 [INFO] root: PluginMagic initialized!
2024-10-19 13:11:25,297 [INFO] commands: Plugin: type ready!
2024-10-19 13:11:25,297 [INFO] root: PluginExample initialized!
2024-10-19 13:11:25,297 [INFO] commands: Plugin: type ready!
2024-10-19 13:11:25,297 [INFO] discord.client: logging in using static token
2024-10-19 13:11:26,348 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: d0acb325ef0cdd959d50037f53a4cadc).
2024-10-19 13:11:28,157 [ERROR] core.dice_engine: Invalid dice expression: 0
2024-10-19 13:11:28,157 [ERROR] root: Error processing dice: Invalid dice expression: 0
2024-10-19 13:12:19,360 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-19 13:12:19,363 [INFO] commands: Loading 2 plugins
2024-10-19 13:12:19,363 [INFO] root: PluginMagic initialized!
2024-10-19 13:12:19,363 [INFO] commands: Plugin: type ready!
2024-10-19 13:12:19,364 [INFO] root: PluginExample initialized!
2024-10-19 13:12:19,364 [INFO] commands: Plugin: type ready!
2024-10-19 13:12:19,364 [INFO] discord.client: logging in using static token
2024-10-19 13:12:20,373 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: ea2889cf5a3186da6964027beba8c522).
2024-10-19 13:13:51,189 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-19 13:13:51,192 [INFO] commands: Loading 2 plugins
2024-10-19 13:13:51,192 [INFO] root: PluginMagic initialized!
2024-10-19 13:13:51,193 [INFO] commands: Plugin: type ready!
2024-10-19 13:13:51,193 [INFO] root: PluginExample initialized!
2024-10-19 13:13:51,193 [INFO] commands: Plugin: type ready!
2024-10-19 13:13:51,193 [INFO] discord.client: logging in using static token
2024-10-19 13:13:52,103 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 0622a1be4f53facd4ede2ff04fc48777).
2024-10-25 12:32:13,241 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-25 12:32:13,246 [INFO] commands: Loading 2 plugins
2024-10-25 12:32:13,247 [INFO] root: PluginMagic initialized!
2024-10-25 12:32:13,247 [INFO] commands: Plugin: type ready!
2024-10-25 12:32:13,247 [INFO] root: PluginExample initialized!
2024-10-25 12:32:13,247 [INFO] commands: Plugin: type ready!
2024-10-25 12:32:13,247 [INFO] discord.client: logging in using static token
2024-10-25 12:32:14,306 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 7aa55ef0fea94f4407c68a627e50f7e3).
2024-10-25 12:32:43,679 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-25 12:32:43,682 [INFO] commands: Loading 2 plugins
2024-10-25 12:32:43,683 [INFO] root: PluginMagic initialized!
2024-10-25 12:32:43,683 [INFO] commands: Plugin: type ready!
2024-10-25 12:32:43,683 [INFO] root: PluginExample initialized!
2024-10-25 12:32:43,683 [INFO] commands: Plugin: type ready!
2024-10-25 12:32:43,683 [INFO] discord.client: logging in using static token
2024-10-25 12:32:44,687 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: f588c1ac0485801a4e59cb32964c7dbe).
2024-10-25 12:32:57,973 [ERROR] root: Error processing dice: 404 Not Found (error code: 10062): Unknown interaction
2024-10-25 12:32:58,219 [ERROR] root: Error processing dice: 404 Not Found (error code: 10062): Unknown interaction
2024-10-25 12:32:58,224 [ERROR] discord.ext.commands.bot: Ignoring exception in command r
Traceback (most recent call last):
  File "/home/rsandrini/Projects/roll-dices-dnd/src/commands/roll_dice.py", line 20, in command_roll_dice
    await send_message(ctx, response)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/core/helper.py", line 45, in send_message
    await ctx.send(message)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
    ...<6 lines>...
    )
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/webhook/async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/app_commands/commands.py", line 858, in _do_call
    return await self._callback(interaction, **params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/Projects/roll-dices-dnd/src/commands/roll_dice.py", line 23, in command_roll_dice
    await ctx.send(f"Exception {e}")
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
    ...<6 lines>...
    )
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/webhook/async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/hybrid.py", line 444, in _invoke_with_namespace
    value = await self._do_call(ctx, ctx.kwargs)  # type: ignore
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/app_commands/commands.py", line 876, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'r' raised an exception: NotFound: 404 Not Found (error code: 10062): Unknown interaction

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/hybrid.py", line 444, in _invoke_with_namespace
    value = await self._do_call(ctx, ctx.kwargs)  # type: ignore
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/app_commands/commands.py", line 876, in _do_call
    raise CommandInvokeError(self, e) from e
discord.ext.commands.errors.HybridCommandError: Hybrid command raised an error: Command 'r' raised an exception: NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-25 12:32:58,408 [ERROR] discord.ext.commands.bot: Ignoring exception in command r
Traceback (most recent call last):
  File "/home/rsandrini/Projects/roll-dices-dnd/src/commands/roll_dice.py", line 20, in command_roll_dice
    await send_message(ctx, response)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/core/helper.py", line 45, in send_message
    await ctx.send(message)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
    ...<6 lines>...
    )
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/webhook/async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/app_commands/commands.py", line 858, in _do_call
    return await self._callback(interaction, **params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/Projects/roll-dices-dnd/src/commands/roll_dice.py", line 23, in command_roll_dice
    await ctx.send(f"Exception {e}")
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
    ...<6 lines>...
    )
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/webhook/async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/hybrid.py", line 444, in _invoke_with_namespace
    value = await self._do_call(ctx, ctx.kwargs)  # type: ignore
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/app_commands/commands.py", line 876, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'r' raised an exception: NotFound: 404 Not Found (error code: 10062): Unknown interaction

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/hybrid.py", line 444, in _invoke_with_namespace
    value = await self._do_call(ctx, ctx.kwargs)  # type: ignore
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/app_commands/commands.py", line 876, in _do_call
    raise CommandInvokeError(self, e) from e
discord.ext.commands.errors.HybridCommandError: Hybrid command raised an error: Command 'r' raised an exception: NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-25 12:34:16,906 [WARNING] discord.gateway: Shard ID None heartbeat blocked for more than 10 seconds.
Loop thread traceback (most recent call last):
  File "/home/rsandrini/Projects/roll-dices-dnd/src/main.py", line 31, in <module>
    bot.run(os.getenv("discord_token"))
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/client.py", line 869, in run
    asyncio.run(runner())
  File "/home/rsandrini/.pyenv/versions/3.13.0/lib/python3.13/asyncio/runners.py", line 194, in run
    return runner.run(main)
  File "/home/rsandrini/.pyenv/versions/3.13.0/lib/python3.13/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
  File "/home/rsandrini/.pyenv/versions/3.13.0/lib/python3.13/asyncio/base_events.py", line 708, in run_until_complete
    self.run_forever()
  File "/home/rsandrini/.pyenv/versions/3.13.0/lib/python3.13/asyncio/base_events.py", line 679, in run_forever
    self._run_once()
  File "/home/rsandrini/.pyenv/versions/3.13.0/lib/python3.13/asyncio/base_events.py", line 2027, in _run_once
    handle._run()
  File "/home/rsandrini/.pyenv/versions/3.13.0/lib/python3.13/asyncio/events.py", line 89, in _run
    self._context.run(self._callback, *self._args)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/bot.py", line 1411, in on_message
    await self.process_commands(message)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/bot.py", line 1408, in process_commands
    await self.invoke(ctx)  # type: ignore
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/commands/roll_dice.py", line 16, in command_roll_dice
    rolls, dice_data, reroll  = await process_input_dice(ctx, dice_data)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/core/dice_engine.py", line 204, in process_input_dice
    await calculate_dice(context, dice_data, dice_positive, dice_negative,
  File "/home/rsandrini/Projects/roll-dices-dnd/src/core/dice_engine.py", line 243, in calculate_dice
    await register_dice_stats(context, roll)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/core/dice_engine.py", line 262, in register_dice_stats
    insert_roll(
  File "/home/rsandrini/Projects/roll-dices-dnd/src/core/stats_db.py", line 33, in insert_roll
    with connect_db():
  File "/home/rsandrini/.pyenv/versions/3.13.0/lib/python3.13/contextlib.py", line 141, in __enter__
    return next(self.gen)
  File "/home/rsandrini/Projects/roll-dices-dnd/src/core/stats_db.py", line 22, in connect_db
    pg_db.connect()
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/peewee.py", line 3263, in connect
    self._state.set_connection(self._connect())
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/peewee.py", line 4001, in _connect
    conn = psycopg2.connect(**params)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)

2024-10-25 12:34:22,964 [ERROR] discord.client: Attempting a reconnect in 1.28s
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/client.py", line 667, in connect
    await self.ws.poll_event()
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/gateway.py", line 625, in poll_event
    await self.received_message(msg.data)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/gateway.py", line 533, in received_message
    await self.send_as_json(beat)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/gateway.py", line 661, in send_as_json
    await self.send(utils._to_json(data))
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/gateway.py", line 657, in send
    await self.socket.send_str(data)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/aiohttp/client_ws.py", line 226, in send_str
    await self._writer.send(data, binary=False, compress=compress)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/aiohttp/http_websocket.py", line 750, in send
    await self._send_frame(message, WSMsgType.TEXT, compress)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/aiohttp/http_websocket.py", line 690, in _send_frame
    self._write(header + mask + message)
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/aiohttp/http_websocket.py", line 723, in _write
    raise ClientConnectionResetError("Cannot write to closing transport")
aiohttp.client_exceptions.ClientConnectionResetError: Cannot write to closing transport
2024-10-25 12:34:24,583 [INFO] discord.gateway: Shard ID None has successfully RESUMED session f588c1ac0485801a4e59cb32964c7dbe.
2024-10-25 12:41:49,440 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-25 12:41:49,446 [INFO] commands: Loading 2 plugins
2024-10-25 12:41:49,447 [INFO] root: PluginMagic initialized!
2024-10-25 12:41:49,447 [INFO] commands: Plugin: type ready!
2024-10-25 12:41:49,448 [INFO] root: PluginExample initialized!
2024-10-25 12:41:49,448 [INFO] commands: Plugin: type ready!
2024-10-25 12:41:49,448 [INFO] discord.client: logging in using static token
2024-10-25 12:41:50,527 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 7febd4530cef5252228c91a684df29b4).
2024-10-25 12:47:30,131 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-25 12:47:30,135 [INFO] commands: Loading 2 plugins
2024-10-25 12:47:30,135 [INFO] root: PluginMagic initialized!
2024-10-25 12:47:30,135 [INFO] commands: Plugin: type ready!
2024-10-25 12:47:30,136 [INFO] root: PluginExample initialized!
2024-10-25 12:47:30,136 [INFO] commands: Plugin: type ready!
2024-10-25 12:47:30,136 [INFO] discord.client: logging in using static token
2024-10-25 12:47:31,251 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 82115f858df0a13000cdb39b6ecaab48).
2024-10-25 12:48:33,853 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-25 12:48:33,859 [INFO] commands: Loading 2 plugins
2024-10-25 12:48:33,860 [INFO] root: PluginMagic initialized!
2024-10-25 12:48:33,860 [INFO] commands: Plugin: type ready!
2024-10-25 12:48:33,860 [INFO] root: PluginExample initialized!
2024-10-25 12:48:33,860 [INFO] commands: Plugin: type ready!
2024-10-25 12:48:33,860 [INFO] discord.client: logging in using static token
2024-10-25 12:48:34,835 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 9906f052e573c6c0c37c18e9f9bb2068).
2024-10-25 13:35:57,232 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-25 13:35:57,238 [INFO] commands: Loading 2 plugins
2024-10-25 13:35:57,239 [INFO] root: PluginMagic initialized!
2024-10-25 13:35:57,239 [INFO] commands: Plugin: type ready!
2024-10-25 13:35:57,239 [INFO] root: PluginExample initialized!
2024-10-25 13:35:57,239 [INFO] commands: Plugin: type ready!
2024-10-25 13:35:57,239 [INFO] discord.client: logging in using static token
2024-10-25 13:35:58,472 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 77d99b65005828d00b1f6aefdbf2b3b8).
2024-10-25 13:42:24,702 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-25 13:42:24,708 [INFO] commands: Loading 2 plugins
2024-10-25 13:42:24,709 [INFO] root: PluginMagic initialized!
2024-10-25 13:42:24,709 [INFO] commands: Plugin: type ready!
2024-10-25 13:42:24,709 [INFO] root: PluginExample initialized!
2024-10-25 13:42:24,709 [INFO] commands: Plugin: type ready!
2024-10-25 13:42:24,709 [INFO] discord.client: logging in using static token
2024-10-25 13:42:25,918 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 932c90a48f5eb9dc08a7ecbc9db902d3).
2024-10-25 13:43:19,651 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-10-25 13:43:19,655 [INFO] commands: Loading 2 plugins
2024-10-25 13:43:19,656 [INFO] root: PluginMagic initialized!
2024-10-25 13:43:19,656 [INFO] commands: Plugin: type ready!
2024-10-25 13:43:19,656 [INFO] root: PluginExample initialized!
2024-10-25 13:43:19,656 [INFO] commands: Plugin: type ready!
2024-10-25 13:43:19,657 [INFO] discord.client: logging in using static token
2024-10-25 13:43:21,032 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 2e2d69ab69dfa76e053dc7820fc48a25).
2024-11-07 11:03:58,763 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-11-07 11:03:58,771 [INFO] commands: Loading 2 plugins
2024-11-07 11:03:58,772 [INFO] root: PluginMagic initialized!
2024-11-07 11:03:58,772 [INFO] commands: Plugin: type ready!
2024-11-07 11:03:58,773 [INFO] root: PluginExample initialized!
2024-11-07 11:03:58,773 [INFO] commands: Plugin: type ready!
2024-11-07 11:03:58,773 [INFO] discord.client: logging in using static token
2024-11-07 11:04:00,199 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 74016290e6111d3e75134b3b0198359e).
2024-11-07 11:05:28,383 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "5" is not found
2024-11-07 11:07:15,375 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-11-07 11:07:15,382 [INFO] commands: Loading 2 plugins
2024-11-07 11:07:15,382 [INFO] root: PluginMagic initialized!
2024-11-07 11:07:15,382 [INFO] commands: Plugin: type ready!
2024-11-07 11:07:15,383 [INFO] root: PluginExample initialized!
2024-11-07 11:07:15,383 [INFO] commands: Plugin: type ready!
2024-11-07 11:07:15,383 [INFO] discord.client: logging in using static token
2024-11-07 11:07:16,350 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 354a3328756a106fd1fb973329b7e04a).
2024-11-07 11:40:11,236 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-11-07 11:40:11,242 [INFO] commands: Loading 2 plugins
2024-11-07 11:40:11,243 [INFO] root: PluginMagic initialized!
2024-11-07 11:40:11,243 [INFO] commands: Plugin: type ready!
2024-11-07 11:40:11,243 [INFO] root: PluginExample initialized!
2024-11-07 11:40:11,243 [INFO] commands: Plugin: type ready!
2024-11-07 11:40:11,244 [INFO] discord.client: logging in using static token
2024-11-07 11:40:12,539 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: c2269be75322334b42a1b65e6474904a).
2024-11-07 11:43:06,746 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-11-07 11:43:06,750 [INFO] commands: Loading 2 plugins
2024-11-07 11:43:06,750 [INFO] root: PluginMagic initialized!
2024-11-07 11:43:06,751 [INFO] commands: Plugin: type ready!
2024-11-07 11:43:06,751 [INFO] root: PluginExample initialized!
2024-11-07 11:43:06,751 [INFO] commands: Plugin: type ready!
2024-11-07 11:43:06,751 [INFO] discord.client: logging in using static token
2024-11-07 11:43:07,674 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 60c5a8da53e21f26c639aab3c3b892a3).
2024-11-07 11:43:36,728 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-11-07 11:43:36,733 [INFO] commands: Loading 2 plugins
2024-11-07 11:43:36,734 [INFO] root: PluginMagic initialized!
2024-11-07 11:43:36,734 [INFO] commands: Plugin: type ready!
2024-11-07 11:43:36,734 [INFO] root: PluginExample initialized!
2024-11-07 11:43:36,734 [INFO] commands: Plugin: type ready!
2024-11-07 11:43:36,735 [INFO] discord.client: logging in using static token
2024-11-07 11:43:37,695 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: aa04c0ba4f450aa8544dc7367571547c).
2024-11-07 11:45:12,358 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "stats" is not found
2024-11-07 11:46:10,402 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "h" is not found
2024-11-07 15:22:33,533 [INFO] discord.gateway: Shard ID None has successfully RESUMED session aa04c0ba4f450aa8544dc7367571547c.
2024-11-07 15:37:55,180 [INFO] discord.gateway: Shard ID None has successfully RESUMED session aa04c0ba4f450aa8544dc7367571547c.
2024-11-29 11:42:35,753 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-11-29 11:42:35,760 [INFO] commands: Loading 2 plugins
2024-11-29 11:42:35,760 [INFO] root: PluginMagic initialized!
2024-11-29 11:42:35,760 [INFO] commands: Plugin: type ready!
2024-11-29 11:42:35,761 [INFO] root: PluginExample initialized!
2024-11-29 11:42:35,761 [INFO] commands: Plugin: type ready!
2024-11-29 11:42:35,761 [INFO] discord.client: logging in using static token
2024-11-29 11:42:37,045 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: e4291185fadc87c7e7a5aaf07aec9519).
2024-11-29 16:45:40,926 [INFO] discord.gateway: Shard ID None session has been invalidated.
2024-11-29 16:45:46,462 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 37477d33a0411abd0fb04741eb829301).
2024-11-29 17:30:32,163 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-11-29 17:30:32,171 [INFO] commands: Loading 2 plugins
2024-11-29 17:30:32,172 [INFO] root: PluginMagic initialized!
2024-11-29 17:30:32,172 [INFO] commands: Plugin: type ready!
2024-11-29 17:30:32,173 [INFO] root: PluginExample initialized!
2024-11-29 17:30:32,173 [INFO] commands: Plugin: type ready!
2024-11-29 17:30:32,173 [INFO] discord.client: logging in using static token
2024-11-29 17:30:33,400 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: d134da6715cf335355decf90a7234291).
2024-11-29 20:19:09,312 [INFO] discord.gateway: Shard ID None has successfully RESUMED session d134da6715cf335355decf90a7234291.
2024-11-29 21:45:42,171 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "i1" is not found
2024-11-29 21:46:17,664 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "remove" is not found
2024-11-29 21:55:08,164 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "i-n" is not found
2024-11-29 21:55:52,610 [ERROR] root: Error processing dice: Invalid remaining data in expression: +5+5+
2024-11-29 22:02:10,589 [INFO] discord.gateway: Shard ID None has successfully RESUMED session d134da6715cf335355decf90a7234291.
2024-11-29 22:09:38,276 [ERROR] root: Error processing dice: Invalid remaining data in expression: +
2024-11-29 22:25:02,533 [ERROR] root: Error processing dice: can only join an iterable
2024-12-06 18:09:30,221 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-12-06 18:09:30,228 [INFO] commands: Loading 2 plugins
2024-12-06 18:09:30,230 [INFO] root: PluginMagic initialized!
2024-12-06 18:09:30,230 [INFO] commands: Plugin: type ready!
2024-12-06 18:09:30,231 [INFO] root: PluginExample initialized!
2024-12-06 18:09:30,231 [INFO] commands: Plugin: type ready!
2024-12-06 18:09:30,231 [INFO] discord.client: logging in using static token
2024-12-06 18:09:31,493 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 3d238f475f74000be5095282abae31c1).
2024-12-06 20:10:40,057 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 3d238f475f74000be5095282abae31c1.
2024-12-06 21:38:37,341 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 3d238f475f74000be5095282abae31c1.
2024-12-06 22:16:00,602 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 3d238f475f74000be5095282abae31c1.
2024-12-06 22:26:29,174 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "R" is not found
2024-12-07 02:05:40,161 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 3d238f475f74000be5095282abae31c1.
2024-12-07 12:35:22,773 [INFO] discord.gateway: Shard ID None session has been invalidated.
2024-12-07 12:35:28,302 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: add9cb1a750b3de1d0ffaba6362bd9d9).
2024-12-07 13:53:09,838 [INFO] discord.gateway: Shard ID None has successfully RESUMED session add9cb1a750b3de1d0ffaba6362bd9d9.
2024-12-07 16:02:20,145 [INFO] discord.gateway: Shard ID None has successfully RESUMED session add9cb1a750b3de1d0ffaba6362bd9d9.
2024-12-07 18:30:16,664 [INFO] discord.gateway: Shard ID None has successfully RESUMED session add9cb1a750b3de1d0ffaba6362bd9d9.
2024-12-07 21:03:54,228 [INFO] discord.gateway: Shard ID None has successfully RESUMED session add9cb1a750b3de1d0ffaba6362bd9d9.
2024-12-07 23:46:37,117 [INFO] discord.gateway: Shard ID None has successfully RESUMED session add9cb1a750b3de1d0ffaba6362bd9d9.
2024-12-08 01:44:12,930 [INFO] discord.gateway: Shard ID None has successfully RESUMED session add9cb1a750b3de1d0ffaba6362bd9d9.
2024-12-08 15:20:41,647 [INFO] discord.gateway: Shard ID None session has been invalidated.
2024-12-08 15:20:47,266 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 97f70d338c4b224411b77c3d70d00df8).
2024-12-08 17:55:36,756 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 97f70d338c4b224411b77c3d70d00df8.
2024-12-11 18:32:20,046 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-12-11 18:32:20,054 [INFO] commands: Loading 2 plugins
2024-12-11 18:32:20,055 [INFO] root: PluginMagic initialized!
2024-12-11 18:32:20,056 [INFO] commands: Plugin: type ready!
2024-12-11 18:32:20,056 [INFO] root: PluginExample initialized!
2024-12-11 18:32:20,056 [INFO] commands: Plugin: type ready!
2024-12-11 18:32:20,056 [INFO] discord.client: logging in using static token
2024-12-11 18:32:21,269 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 153a3dda8dba522848a816a6163bf1be).
2024-12-11 20:30:22,767 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "ilimpar" is not found
2024-12-11 20:31:20,526 [ERROR] root: Error rolling initiative: Invalid remaining data in expression: +
2024-12-11 20:57:31,961 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 153a3dda8dba522848a816a6163bf1be.
2024-12-11 20:59:26,135 [ERROR] root: Error rolling initiative: invalid syntax (<string>, line 0)
2024-12-11 21:21:20,282 [ERROR] core.dice_engine: Invalid dice expression: f321334123
2024-12-11 21:21:20,287 [ERROR] root: Error processing dice: Invalid dice expression: f321334123
2024-12-11 21:56:21,683 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "magiic" is not found
2024-12-13 18:53:39,299 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-12-13 18:53:39,306 [INFO] commands: Loading 2 plugins
2024-12-13 18:53:39,308 [INFO] root: PluginMagic initialized!
2024-12-13 18:53:39,308 [INFO] commands: Plugin: type ready!
2024-12-13 18:53:39,308 [INFO] root: PluginExample initialized!
2024-12-13 18:53:39,308 [INFO] commands: Plugin: type ready!
2024-12-13 18:53:39,308 [INFO] discord.client: logging in using static token
2024-12-13 18:53:40,851 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 6d427bcf7332747be0646d6f5cfef721).
2024-12-13 22:08:52,359 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 6d427bcf7332747be0646d6f5cfef721.
2024-12-13 22:22:44,094 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 6d427bcf7332747be0646d6f5cfef721.
2024-12-13 23:12:26,017 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 6d427bcf7332747be0646d6f5cfef721.
2024-12-20 19:04:06,702 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2024-12-20 19:04:06,710 [INFO] commands: Loading 2 plugins
2024-12-20 19:04:06,712 [INFO] root: PluginMagic initialized!
2024-12-20 19:04:06,712 [INFO] commands: Plugin: type ready!
2024-12-20 19:04:06,712 [INFO] root: PluginExample initialized!
2024-12-20 19:04:06,712 [INFO] commands: Plugin: type ready!
2024-12-20 19:04:06,713 [INFO] discord.client: logging in using static token
2024-12-20 19:04:07,942 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 3189833e5a39501505decd57b7adfe02).
2024-12-20 21:11:25,098 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 3189833e5a39501505decd57b7adfe02.
2024-12-20 21:24:13,530 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "magic]" is not found
2024-12-20 21:33:01,092 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "ihelp" is not found
2024-12-20 21:51:40,308 [ERROR] core.dice_engine: Invalid dice expression: 2tintin
2024-12-20 21:51:40,314 [ERROR] root: Error processing dice: Invalid dice expression: 2tintin
2024-12-20 22:24:17,944 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "n" is not found
2024-12-20 22:24:39,407 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "n" is not found
2024-12-20 22:31:51,996 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "MAGIC" is not found
2025-04-22 12:23:11,636 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:23:22,771 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:23:22,778 [INFO] commands: Loading 3 plugins
2025-04-22 12:23:22,779 [INFO] root: PluginNewChar initialized!
2025-04-22 12:23:22,779 [INFO] commands: Plugin: type ready!
2025-04-22 12:23:22,781 [INFO] root: PluginMagic initialized!
2025-04-22 12:23:22,781 [INFO] commands: Plugin: type ready!
2025-04-22 12:23:22,781 [INFO] root: PluginExample initialized!
2025-04-22 12:23:22,781 [INFO] commands: Plugin: type ready!
2025-04-22 12:23:22,781 [INFO] discord.client: logging in using static token
2025-04-22 12:23:23,797 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 31dc87a5057cd0a70ba7aec11ee7f61f).
2025-04-22 12:24:47,714 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "!r" is not found
2025-04-22 12:25:04,274 [ERROR] core.dice_engine: Invalid dice expression: new-char
2025-04-22 12:25:04,275 [ERROR] root: Error processing dice: Invalid dice expression: new-char
2025-04-22 12:25:48,798 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:25:48,804 [INFO] commands: Loading 3 plugins
2025-04-22 12:25:48,804 [INFO] root: PluginNewChar initialized!
2025-04-22 12:25:48,804 [INFO] commands: Plugin: type ready!
2025-04-22 12:25:48,805 [INFO] root: PluginMagic initialized!
2025-04-22 12:25:48,805 [INFO] commands: Plugin: type ready!
2025-04-22 12:25:48,805 [INFO] root: PluginExample initialized!
2025-04-22 12:25:48,805 [INFO] commands: Plugin: type ready!
2025-04-22 12:25:48,806 [INFO] discord.client: logging in using static token
2025-04-22 12:25:49,783 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 425f2867e534501b0ce509a93a26f36a).
2025-04-22 12:27:35,361 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:27:35,365 [INFO] commands: Loading 3 plugins
2025-04-22 12:27:35,365 [INFO] root: PluginNewChar initialized!
2025-04-22 12:27:35,365 [INFO] commands: Plugin: type ready!
2025-04-22 12:27:35,366 [INFO] root: PluginMagic initialized!
2025-04-22 12:27:35,366 [INFO] commands: Plugin: type ready!
2025-04-22 12:27:35,366 [INFO] root: PluginExample initialized!
2025-04-22 12:27:35,366 [INFO] commands: Plugin: type ready!
2025-04-22 12:27:35,367 [INFO] discord.client: logging in using static token
2025-04-22 12:27:36,352 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: ca89dd0a4fba6d6f0f62b68f4e1a2225).
2025-04-22 12:29:23,331 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:29:23,339 [INFO] commands: Loading 3 plugins
2025-04-22 12:29:23,339 [INFO] root: PluginNewChar initialized!
2025-04-22 12:29:23,339 [INFO] commands: Plugin: type ready!
2025-04-22 12:29:23,340 [INFO] root: PluginMagic initialized!
2025-04-22 12:29:23,340 [INFO] commands: Plugin: type ready!
2025-04-22 12:29:23,340 [INFO] root: PluginExample initialized!
2025-04-22 12:29:23,340 [INFO] commands: Plugin: type ready!
2025-04-22 12:29:23,340 [INFO] discord.client: logging in using static token
2025-04-22 12:29:24,472 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 784fb1bcef0c8d45a014dc15abe71cce).
2025-04-22 12:29:38,686 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:29:38,693 [INFO] commands: Loading 3 plugins
2025-04-22 12:29:38,693 [INFO] root: PluginNewChar initialized!
2025-04-22 12:29:38,693 [INFO] commands: Plugin: type ready!
2025-04-22 12:29:38,694 [INFO] root: PluginMagic initialized!
2025-04-22 12:29:38,694 [INFO] commands: Plugin: type ready!
2025-04-22 12:29:38,694 [INFO] root: PluginExample initialized!
2025-04-22 12:29:38,694 [INFO] commands: Plugin: type ready!
2025-04-22 12:29:38,694 [INFO] discord.client: logging in using static token
2025-04-22 12:29:40,041 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 5946fe29f0a896c7da59ada094f37698).
2025-04-22 12:31:44,544 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:31:44,550 [INFO] commands: Loading 3 plugins
2025-04-22 12:31:44,551 [INFO] root: PluginNewChar initialized!
2025-04-22 12:31:44,551 [INFO] commands: Plugin: type ready!
2025-04-22 12:31:44,551 [INFO] root: PluginMagic initialized!
2025-04-22 12:31:44,551 [INFO] commands: Plugin: type ready!
2025-04-22 12:31:44,551 [INFO] root: PluginExample initialized!
2025-04-22 12:31:44,551 [INFO] commands: Plugin: type ready!
2025-04-22 12:31:44,552 [INFO] discord.client: logging in using static token
2025-04-22 12:31:45,682 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 8d45cf7aee8912967a720819de96f129).
2025-04-22 12:32:34,341 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:32:34,347 [INFO] commands: Loading 3 plugins
2025-04-22 12:32:34,347 [INFO] root: PluginNewChar initialized!
2025-04-22 12:32:34,348 [INFO] commands: Plugin: type ready!
2025-04-22 12:32:34,348 [INFO] root: PluginMagic initialized!
2025-04-22 12:32:34,348 [INFO] commands: Plugin: type ready!
2025-04-22 12:32:34,348 [INFO] root: PluginExample initialized!
2025-04-22 12:32:34,348 [INFO] commands: Plugin: type ready!
2025-04-22 12:32:34,349 [INFO] discord.client: logging in using static token
2025-04-22 12:32:35,411 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 338e59c84e98381a4dc68607b080a607).
2025-04-22 12:41:53,328 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:41:53,336 [INFO] commands: Loading 3 plugins
2025-04-22 12:41:53,337 [INFO] root: PluginNewChar initialized!
2025-04-22 12:41:53,337 [INFO] commands: Plugin: type ready!
2025-04-22 12:41:53,337 [INFO] root: PluginMagic initialized!
2025-04-22 12:41:53,337 [INFO] commands: Plugin: type ready!
2025-04-22 12:41:53,337 [INFO] root: PluginExample initialized!
2025-04-22 12:41:53,337 [INFO] commands: Plugin: type ready!
2025-04-22 12:41:53,338 [INFO] discord.client: logging in using static token
2025-04-22 12:41:54,489 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1c8aa3e90b5fce196f5d09d5e7b343b3).
2025-04-22 12:42:07,383 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:42:07,388 [INFO] commands: Loading 3 plugins
2025-04-22 12:42:07,388 [INFO] root: PluginNewChar initialized!
2025-04-22 12:42:07,388 [INFO] commands: Plugin: type ready!
2025-04-22 12:42:07,389 [INFO] root: PluginMagic initialized!
2025-04-22 12:42:07,389 [INFO] commands: Plugin: type ready!
2025-04-22 12:42:07,389 [INFO] root: PluginExample initialized!
2025-04-22 12:42:07,389 [INFO] commands: Plugin: type ready!
2025-04-22 12:42:07,390 [INFO] discord.client: logging in using static token
2025-04-22 12:42:08,347 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: c0cf96d6355233a3c610774da7217b5d).
2025-04-22 12:43:38,939 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:43:38,947 [INFO] commands: Loading 3 plugins
2025-04-22 12:43:38,947 [INFO] root: PluginNewChar initialized!
2025-04-22 12:43:38,947 [INFO] commands: Plugin: type ready!
2025-04-22 12:43:38,947 [INFO] root: PluginMagic initialized!
2025-04-22 12:43:38,947 [INFO] commands: Plugin: type ready!
2025-04-22 12:43:38,948 [INFO] root: PluginExample initialized!
2025-04-22 12:43:38,948 [INFO] commands: Plugin: type ready!
2025-04-22 12:43:38,948 [INFO] discord.client: logging in using static token
2025-04-22 12:43:39,951 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 4e96c85dca591881739e4d0c18c9fe2d).
2025-04-22 12:48:22,904 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:48:22,910 [INFO] commands: Loading 3 plugins
2025-04-22 12:48:22,910 [INFO] root: PluginNewChar initialized!
2025-04-22 12:48:22,910 [INFO] commands: Plugin: type ready!
2025-04-22 12:48:22,911 [INFO] root: PluginMagic initialized!
2025-04-22 12:48:22,911 [INFO] commands: Plugin: type ready!
2025-04-22 12:48:22,911 [INFO] root: PluginExample initialized!
2025-04-22 12:48:22,911 [INFO] commands: Plugin: type ready!
2025-04-22 12:48:22,911 [INFO] discord.client: logging in using static token
2025-04-22 12:48:23,933 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1da0726cb4f30452718d03f41b05eb8e).
2025-04-22 12:52:35,645 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:52:35,652 [INFO] commands: Loading 3 plugins
2025-04-22 12:52:35,652 [INFO] root: PluginNewChar initialized!
2025-04-22 12:52:35,652 [INFO] commands: Plugin: type ready!
2025-04-22 12:52:35,652 [INFO] root: PluginMagic initialized!
2025-04-22 12:52:35,653 [INFO] commands: Plugin: type ready!
2025-04-22 12:52:35,653 [INFO] root: PluginExample initialized!
2025-04-22 12:52:35,653 [INFO] commands: Plugin: type ready!
2025-04-22 12:52:35,653 [INFO] discord.client: logging in using static token
2025-04-22 12:52:36,567 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 6b8292ffa62e0152a1618c2ce5e41e84).
2025-04-22 12:54:38,245 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:54:38,249 [INFO] commands: Loading 3 plugins
2025-04-22 12:54:38,249 [INFO] root: PluginNewChar initialized!
2025-04-22 12:54:38,250 [INFO] commands: Plugin: type ready!
2025-04-22 12:54:38,250 [INFO] root: PluginMagic initialized!
2025-04-22 12:54:38,250 [INFO] commands: Plugin: type ready!
2025-04-22 12:54:38,250 [INFO] root: PluginExample initialized!
2025-04-22 12:54:38,250 [INFO] commands: Plugin: type ready!
2025-04-22 12:54:38,251 [INFO] discord.client: logging in using static token
2025-04-22 12:54:39,474 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: e404dcb8bf264419b77f9023b3265966).
2025-04-22 12:54:41,290 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "!nenw-char" is not found
2025-04-22 12:55:36,603 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 12:55:36,608 [INFO] commands: Loading 3 plugins
2025-04-22 12:55:36,608 [INFO] root: PluginNewChar initialized!
2025-04-22 12:55:36,609 [INFO] commands: Plugin: type ready!
2025-04-22 12:55:36,609 [INFO] root: PluginMagic initialized!
2025-04-22 12:55:36,609 [INFO] commands: Plugin: type ready!
2025-04-22 12:55:36,609 [INFO] root: PluginExample initialized!
2025-04-22 12:55:36,609 [INFO] commands: Plugin: type ready!
2025-04-22 12:55:36,610 [INFO] discord.client: logging in using static token
2025-04-22 12:55:37,739 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 4564aabf4c65daaceb380bfa80775589).
2025-04-22 13:22:09,522 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 4564aabf4c65daaceb380bfa80775589.
2025-04-22 15:20:14,620 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 4564aabf4c65daaceb380bfa80775589.
2025-04-22 15:46:51,727 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 4564aabf4c65daaceb380bfa80775589.
2025-04-22 18:30:49,957 [INFO] discord.gateway: Shard ID None has successfully RESUMED session 4564aabf4c65daaceb380bfa80775589.
2025-04-22 19:56:09,567 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 19:56:09,575 [INFO] commands: Loading 3 plugins
2025-04-22 19:56:09,575 [INFO] root: PluginNewChar initialized!
2025-04-22 19:56:09,575 [INFO] commands: Plugin: type ready!
2025-04-22 19:56:09,575 [INFO] root: PluginMagic initialized!
2025-04-22 19:56:09,576 [INFO] commands: Plugin: type ready!
2025-04-22 19:56:09,576 [INFO] root: PluginExample initialized!
2025-04-22 19:56:09,576 [INFO] commands: Plugin: type ready!
2025-04-22 19:56:09,576 [INFO] discord.client: logging in using static token
2025-04-22 19:56:10,775 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 65b7f09f49802c5704ad620081d06c15).
2025-04-22 19:57:32,177 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 19:57:32,186 [INFO] commands: Loading 3 plugins
2025-04-22 19:57:32,186 [INFO] root: PluginNewChar initialized!
2025-04-22 19:57:32,186 [INFO] commands: Plugin: type ready!
2025-04-22 19:57:32,186 [INFO] root: PluginMagic initialized!
2025-04-22 19:57:32,187 [INFO] commands: Plugin: type ready!
2025-04-22 19:57:32,187 [INFO] root: PluginExample initialized!
2025-04-22 19:57:32,187 [INFO] commands: Plugin: type ready!
2025-04-22 19:57:32,187 [INFO] discord.client: logging in using static token
2025-04-22 19:57:33,294 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: fb620868f279b8485aa7767348f23457).
2025-04-22 19:57:38,619 [ERROR] root: An error occurred while generating the roll text: 'list' object has no attribute 'total'
2025-04-22 19:57:39,159 [ERROR] discord.ext.commands.bot: Ignoring exception in command new-char
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/new_char/plugin_new_char.py", line 41, in new_char_roll
    await context.send(
        await get_new_char_roll_text(context, roll))
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/context.py", line 1039, in send
    return await super().send(
           ^^^^^^^^^^^^^^^^^^^
    ...<16 lines>...
    )  # type: ignore # The overloads don't support Optional but the implementation does
    ^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/abc.py", line 1618, in send
    data = await state.http.send_message(channel.id, params=params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50006): Cannot send an empty message

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: HTTPException: 400 Bad Request (error code: 50006): Cannot send an empty message
2025-04-22 19:58:11,452 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 19:58:11,456 [INFO] commands: Loading 3 plugins
2025-04-22 19:58:11,456 [INFO] root: PluginNewChar initialized!
2025-04-22 19:58:11,456 [INFO] commands: Plugin: type ready!
2025-04-22 19:58:11,457 [INFO] root: PluginMagic initialized!
2025-04-22 19:58:11,457 [INFO] commands: Plugin: type ready!
2025-04-22 19:58:11,457 [INFO] root: PluginExample initialized!
2025-04-22 19:58:11,457 [INFO] commands: Plugin: type ready!
2025-04-22 19:58:11,457 [INFO] discord.client: logging in using static token
2025-04-22 19:58:12,421 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 76dc3894db8a2abfe201f9cc500744a4).
2025-04-22 19:58:19,384 [ERROR] root: An error occurred while generating the roll text: 'list' object has no attribute 'total'
2025-04-22 19:58:19,886 [ERROR] discord.ext.commands.bot: Ignoring exception in command new-char
Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/Projects/roll-dices-dnd/src/plugins/new_char/plugin_new_char.py", line 40, in new_char_roll
    await context.send(
        await get_new_char_roll_text(context, roll))
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/context.py", line 1039, in send
    return await super().send(
           ^^^^^^^^^^^^^^^^^^^
    ...<16 lines>...
    )  # type: ignore # The overloads don't support Optional but the implementation does
    ^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/abc.py", line 1618, in send
    data = await state.http.send_message(channel.id, params=params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50006): Cannot send an empty message

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rsandrini/.cache/pypoetry/virtualenvs/roll_dice_discord_bot-8uYDqlfy-py3.13/lib/python3.13/site-packages/discord/ext/commands/core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: HTTPException: 400 Bad Request (error code: 50006): Cannot send an empty message
2025-04-22 20:02:32,183 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:02:32,187 [INFO] commands: Loading 3 plugins
2025-04-22 20:02:32,188 [INFO] root: PluginNewChar initialized!
2025-04-22 20:02:32,188 [INFO] commands: Plugin: type ready!
2025-04-22 20:02:32,188 [INFO] root: PluginMagic initialized!
2025-04-22 20:02:32,188 [INFO] commands: Plugin: type ready!
2025-04-22 20:02:32,188 [INFO] root: PluginExample initialized!
2025-04-22 20:02:32,189 [INFO] commands: Plugin: type ready!
2025-04-22 20:02:32,189 [INFO] discord.client: logging in using static token
2025-04-22 20:02:33,391 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: b876c21b59bb62c7886a8eb1c9a213b1).
2025-04-22 20:02:44,161 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:02:44,167 [INFO] commands: Loading 3 plugins
2025-04-22 20:02:44,167 [INFO] root: PluginNewChar initialized!
2025-04-22 20:02:44,168 [INFO] commands: Plugin: type ready!
2025-04-22 20:02:44,168 [INFO] root: PluginMagic initialized!
2025-04-22 20:02:44,168 [INFO] commands: Plugin: type ready!
2025-04-22 20:02:44,168 [INFO] root: PluginExample initialized!
2025-04-22 20:02:44,168 [INFO] commands: Plugin: type ready!
2025-04-22 20:02:44,169 [INFO] discord.client: logging in using static token
2025-04-22 20:02:45,166 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: e2b14204be33480577e0ffa503bb0e21).
2025-04-22 20:02:52,798 [ERROR] discord.ext.commands.bot: Ignoring exception in command None
discord.ext.commands.errors.CommandNotFound: Command "!new-char" is not found
2025-04-22 20:03:45,564 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:03:45,572 [INFO] commands: Loading 3 plugins
2025-04-22 20:03:45,572 [INFO] root: PluginNewChar initialized!
2025-04-22 20:03:45,572 [INFO] commands: Plugin: type ready!
2025-04-22 20:03:45,573 [INFO] root: PluginMagic initialized!
2025-04-22 20:03:45,573 [INFO] commands: Plugin: type ready!
2025-04-22 20:03:45,573 [INFO] root: PluginExample initialized!
2025-04-22 20:03:45,573 [INFO] commands: Plugin: type ready!
2025-04-22 20:03:45,574 [INFO] discord.client: logging in using static token
2025-04-22 20:03:46,743 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: c9b4ea925d739e4e86f89bdf120d167b).
2025-04-22 20:06:29,266 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:06:29,274 [INFO] commands: Loading 3 plugins
2025-04-22 20:06:29,274 [INFO] root: PluginNewChar initialized!
2025-04-22 20:06:29,274 [INFO] commands: Plugin: type ready!
2025-04-22 20:06:29,275 [INFO] root: PluginMagic initialized!
2025-04-22 20:06:29,275 [INFO] commands: Plugin: type ready!
2025-04-22 20:06:29,275 [INFO] root: PluginExample initialized!
2025-04-22 20:06:29,275 [INFO] commands: Plugin: type ready!
2025-04-22 20:06:29,276 [INFO] discord.client: logging in using static token
2025-04-22 20:06:30,281 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: d2c428d117395981dac5cd4c1b84ada5).
2025-04-22 20:07:29,285 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:07:29,292 [INFO] commands: Loading 3 plugins
2025-04-22 20:07:29,292 [INFO] root: PluginNewChar initialized!
2025-04-22 20:07:29,292 [INFO] commands: Plugin: type ready!
2025-04-22 20:07:29,293 [INFO] root: PluginMagic initialized!
2025-04-22 20:07:29,293 [INFO] commands: Plugin: type ready!
2025-04-22 20:07:29,293 [INFO] root: PluginExample initialized!
2025-04-22 20:07:29,293 [INFO] commands: Plugin: type ready!
2025-04-22 20:07:29,294 [INFO] discord.client: logging in using static token
2025-04-22 20:07:30,246 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 2fe3ad5eefbb127582e6df8f2271200b).
2025-04-22 20:09:18,587 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:09:18,595 [INFO] commands: Loading 3 plugins
2025-04-22 20:09:18,595 [INFO] root: PluginNewChar initialized!
2025-04-22 20:09:18,595 [INFO] commands: Plugin: type ready!
2025-04-22 20:09:18,595 [INFO] root: PluginMagic initialized!
2025-04-22 20:09:18,596 [INFO] commands: Plugin: type ready!
2025-04-22 20:09:18,596 [INFO] root: PluginExample initialized!
2025-04-22 20:09:18,596 [INFO] commands: Plugin: type ready!
2025-04-22 20:09:18,596 [INFO] discord.client: logging in using static token
2025-04-22 20:09:19,823 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 4471010adcbcdbf0f2fddbcab69cd551).
2025-04-22 20:10:32,921 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:10:32,928 [INFO] commands: Loading 3 plugins
2025-04-22 20:10:32,929 [INFO] root: PluginNewChar initialized!
2025-04-22 20:10:32,929 [INFO] commands: Plugin: type ready!
2025-04-22 20:10:32,929 [INFO] root: PluginMagic initialized!
2025-04-22 20:10:32,929 [INFO] commands: Plugin: type ready!
2025-04-22 20:10:32,930 [INFO] root: PluginExample initialized!
2025-04-22 20:10:32,930 [INFO] commands: Plugin: type ready!
2025-04-22 20:10:32,930 [INFO] discord.client: logging in using static token
2025-04-22 20:10:33,877 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 166be1bed3abbdad587b93725be0a703).
2025-04-22 20:11:04,783 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:11:04,787 [INFO] commands: Loading 3 plugins
2025-04-22 20:11:04,788 [INFO] root: PluginNewChar initialized!
2025-04-22 20:11:04,788 [INFO] commands: Plugin: type ready!
2025-04-22 20:11:04,788 [INFO] root: PluginMagic initialized!
2025-04-22 20:11:04,788 [INFO] commands: Plugin: type ready!
2025-04-22 20:11:04,789 [INFO] root: PluginExample initialized!
2025-04-22 20:11:04,789 [INFO] commands: Plugin: type ready!
2025-04-22 20:11:04,789 [INFO] discord.client: logging in using static token
2025-04-22 20:11:05,773 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 982e13f90c6ce28274e35e0fdee37590).
2025-04-22 20:14:16,271 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:14:16,278 [INFO] commands: Loading 3 plugins
2025-04-22 20:14:16,279 [INFO] root: PluginNewChar initialized!
2025-04-22 20:14:16,279 [INFO] commands: Plugin: type ready!
2025-04-22 20:14:16,279 [INFO] root: PluginMagic initialized!
2025-04-22 20:14:16,279 [INFO] commands: Plugin: type ready!
2025-04-22 20:14:16,279 [INFO] root: PluginExample initialized!
2025-04-22 20:14:16,279 [INFO] commands: Plugin: type ready!
2025-04-22 20:14:16,280 [INFO] discord.client: logging in using static token
2025-04-22 20:14:17,268 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 29c6f7621a71d0ae0dd959d0bbd08b56).
2025-04-22 20:14:35,044 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:14:35,051 [INFO] commands: Loading 3 plugins
2025-04-22 20:14:35,051 [INFO] root: PluginNewChar initialized!
2025-04-22 20:14:35,051 [INFO] commands: Plugin: type ready!
2025-04-22 20:14:35,052 [INFO] root: PluginMagic initialized!
2025-04-22 20:14:35,052 [INFO] commands: Plugin: type ready!
2025-04-22 20:14:35,052 [INFO] root: PluginExample initialized!
2025-04-22 20:14:35,052 [INFO] commands: Plugin: type ready!
2025-04-22 20:14:35,052 [INFO] discord.client: logging in using static token
2025-04-22 20:14:36,144 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: bfb89bf130c4d08b346ec5615cc9ef05).
2025-04-22 20:14:56,767 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:14:56,774 [INFO] commands: Loading 3 plugins
2025-04-22 20:14:56,774 [INFO] root: PluginNewChar initialized!
2025-04-22 20:14:56,774 [INFO] commands: Plugin: type ready!
2025-04-22 20:14:56,775 [INFO] root: PluginMagic initialized!
2025-04-22 20:14:56,775 [INFO] commands: Plugin: type ready!
2025-04-22 20:14:56,775 [INFO] root: PluginExample initialized!
2025-04-22 20:14:56,775 [INFO] commands: Plugin: type ready!
2025-04-22 20:14:56,775 [INFO] discord.client: logging in using static token
2025-04-22 20:14:57,685 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: c6aedf627331b724fa80716538740782).
2025-04-22 20:15:16,722 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:15:16,727 [INFO] commands: Loading 3 plugins
2025-04-22 20:15:16,728 [INFO] root: PluginNewChar initialized!
2025-04-22 20:15:16,728 [INFO] commands: Plugin: type ready!
2025-04-22 20:15:16,728 [INFO] root: PluginMagic initialized!
2025-04-22 20:15:16,728 [INFO] commands: Plugin: type ready!
2025-04-22 20:15:16,728 [INFO] root: PluginExample initialized!
2025-04-22 20:15:16,729 [INFO] commands: Plugin: type ready!
2025-04-22 20:15:16,729 [INFO] discord.client: logging in using static token
2025-04-22 20:15:17,613 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: cf93d37e72b97b378417393a5c33273a).
2025-04-22 20:18:35,026 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:18:35,033 [INFO] commands: Loading 3 plugins
2025-04-22 20:18:35,034 [INFO] root: PluginNewChar initialized!
2025-04-22 20:18:35,034 [INFO] commands: Plugin: type ready!
2025-04-22 20:18:35,034 [INFO] root: PluginMagic initialized!
2025-04-22 20:18:35,034 [INFO] commands: Plugin: type ready!
2025-04-22 20:18:35,035 [INFO] root: PluginExample initialized!
2025-04-22 20:18:35,035 [INFO] commands: Plugin: type ready!
2025-04-22 20:18:35,035 [INFO] discord.client: logging in using static token
2025-04-22 20:18:35,989 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: b94545f27cdf546385b8418c7956fc21).
2025-04-22 20:19:10,882 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:19:10,889 [INFO] commands: Loading 3 plugins
2025-04-22 20:19:10,890 [INFO] root: PluginNewChar initialized!
2025-04-22 20:19:10,890 [INFO] commands: Plugin: type ready!
2025-04-22 20:19:10,890 [INFO] root: PluginMagic initialized!
2025-04-22 20:19:10,890 [INFO] commands: Plugin: type ready!
2025-04-22 20:19:10,890 [INFO] root: PluginExample initialized!
2025-04-22 20:19:10,890 [INFO] commands: Plugin: type ready!
2025-04-22 20:19:10,891 [INFO] discord.client: logging in using static token
2025-04-22 20:19:11,803 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 83a005cea04570084356bdd865786d8a).
2025-04-22 20:20:22,321 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:20:22,328 [INFO] commands: Loading 3 plugins
2025-04-22 20:20:22,328 [INFO] root: PluginNewChar initialized!
2025-04-22 20:20:22,328 [INFO] commands: Plugin: type ready!
2025-04-22 20:20:22,329 [INFO] root: PluginMagic initialized!
2025-04-22 20:20:22,329 [INFO] commands: Plugin: type ready!
2025-04-22 20:20:22,329 [INFO] root: PluginExample initialized!
2025-04-22 20:20:22,329 [INFO] commands: Plugin: type ready!
2025-04-22 20:20:22,330 [INFO] discord.client: logging in using static token
2025-04-22 20:20:23,904 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 937034890f0fc557ae702a193d6f4f74).
2025-04-22 20:21:36,817 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:21:36,826 [INFO] commands: Loading 3 plugins
2025-04-22 20:21:36,826 [INFO] root: PluginNewChar initialized!
2025-04-22 20:21:36,826 [INFO] commands: Plugin: type ready!
2025-04-22 20:21:36,826 [INFO] root: PluginMagic initialized!
2025-04-22 20:21:36,827 [INFO] commands: Plugin: type ready!
2025-04-22 20:21:36,827 [INFO] root: PluginExample initialized!
2025-04-22 20:21:36,827 [INFO] commands: Plugin: type ready!
2025-04-22 20:21:36,827 [INFO] discord.client: logging in using static token
2025-04-22 20:21:37,729 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 63651beae7d0bc543600836c3f32146f).
2025-04-22 20:22:11,661 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:22:11,667 [INFO] commands: Loading 3 plugins
2025-04-22 20:22:11,668 [INFO] root: PluginNewChar initialized!
2025-04-22 20:22:11,668 [INFO] commands: Plugin: type ready!
2025-04-22 20:22:11,668 [INFO] root: PluginMagic initialized!
2025-04-22 20:22:11,668 [INFO] commands: Plugin: type ready!
2025-04-22 20:22:11,668 [INFO] root: PluginExample initialized!
2025-04-22 20:22:11,669 [INFO] commands: Plugin: type ready!
2025-04-22 20:22:11,669 [INFO] discord.client: logging in using static token
2025-04-22 20:22:12,606 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: f0cf6afc310369e4e616b89e6f84f561).
2025-04-22 20:23:31,331 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:23:31,338 [INFO] commands: Loading 3 plugins
2025-04-22 20:23:31,338 [INFO] root: PluginNewChar initialized!
2025-04-22 20:23:31,338 [INFO] commands: Plugin: type ready!
2025-04-22 20:23:31,339 [INFO] root: PluginMagic initialized!
2025-04-22 20:23:31,339 [INFO] commands: Plugin: type ready!
2025-04-22 20:23:31,339 [INFO] root: PluginExample initialized!
2025-04-22 20:23:31,339 [INFO] commands: Plugin: type ready!
2025-04-22 20:23:31,339 [INFO] discord.client: logging in using static token
2025-04-22 20:23:32,261 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: da27d4e0ad1310eda4a91add5c9e015f).
2025-04-22 20:24:15,425 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:24:15,432 [INFO] commands: Loading 3 plugins
2025-04-22 20:24:15,432 [INFO] root: PluginNewChar initialized!
2025-04-22 20:24:15,432 [INFO] commands: Plugin: type ready!
2025-04-22 20:24:15,433 [INFO] root: PluginMagic initialized!
2025-04-22 20:24:15,433 [INFO] commands: Plugin: type ready!
2025-04-22 20:24:15,433 [INFO] root: PluginExample initialized!
2025-04-22 20:24:15,433 [INFO] commands: Plugin: type ready!
2025-04-22 20:24:15,434 [INFO] discord.client: logging in using static token
2025-04-22 20:24:16,672 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 3d87910ac094263fdbe324baec78d5c2).
2025-04-22 20:24:45,633 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:24:45,641 [INFO] commands: Loading 3 plugins
2025-04-22 20:24:45,641 [INFO] root: PluginNewChar initialized!
2025-04-22 20:24:45,641 [INFO] commands: Plugin: type ready!
2025-04-22 20:24:45,642 [INFO] root: PluginMagic initialized!
2025-04-22 20:24:45,642 [INFO] commands: Plugin: type ready!
2025-04-22 20:24:45,642 [INFO] root: PluginExample initialized!
2025-04-22 20:24:45,642 [INFO] commands: Plugin: type ready!
2025-04-22 20:24:45,642 [INFO] discord.client: logging in using static token
2025-04-22 20:24:46,676 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: aff94e02f8f6e49327953e72d25ed778).
2025-04-22 20:26:44,661 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:26:44,668 [INFO] commands: Loading 3 plugins
2025-04-22 20:26:44,668 [INFO] root: PluginNewChar initialized!
2025-04-22 20:26:44,668 [INFO] commands: Plugin: type ready!
2025-04-22 20:26:44,669 [INFO] root: PluginMagic initialized!
2025-04-22 20:26:44,669 [INFO] commands: Plugin: type ready!
2025-04-22 20:26:44,669 [INFO] root: PluginExample initialized!
2025-04-22 20:26:44,669 [INFO] commands: Plugin: type ready!
2025-04-22 20:26:44,670 [INFO] discord.client: logging in using static token
2025-04-22 20:26:45,767 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: b8464239c824da790d477e3c1019c56c).
2025-04-22 20:27:16,814 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:27:16,821 [INFO] commands: Loading 3 plugins
2025-04-22 20:27:16,821 [INFO] root: PluginNewChar initialized!
2025-04-22 20:27:16,821 [INFO] commands: Plugin: type ready!
2025-04-22 20:27:16,822 [INFO] root: PluginMagic initialized!
2025-04-22 20:27:16,822 [INFO] commands: Plugin: type ready!
2025-04-22 20:27:16,822 [INFO] root: PluginExample initialized!
2025-04-22 20:27:16,822 [INFO] commands: Plugin: type ready!
2025-04-22 20:27:16,822 [INFO] discord.client: logging in using static token
2025-04-22 20:27:18,034 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 43c9c3f1c552c442870a2a192b794a90).
2025-04-22 20:29:13,860 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:29:13,867 [INFO] commands: Loading 3 plugins
2025-04-22 20:29:13,868 [INFO] root: PluginNewChar initialized!
2025-04-22 20:29:13,868 [INFO] commands: Plugin: type ready!
2025-04-22 20:29:13,868 [INFO] root: PluginMagic initialized!
2025-04-22 20:29:13,868 [INFO] commands: Plugin: type ready!
2025-04-22 20:29:13,868 [INFO] root: PluginExample initialized!
2025-04-22 20:29:13,869 [INFO] commands: Plugin: type ready!
2025-04-22 20:29:13,869 [INFO] discord.client: logging in using static token
2025-04-22 20:29:15,059 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: c1a742393360329c9edca2804d0cce72).
2025-04-22 20:31:48,943 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:31:48,947 [INFO] commands: Loading 3 plugins
2025-04-22 20:31:48,948 [INFO] root: PluginNewChar initialized!
2025-04-22 20:31:48,948 [INFO] commands: Plugin: type ready!
2025-04-22 20:31:48,948 [INFO] root: PluginMagic initialized!
2025-04-22 20:31:48,948 [INFO] commands: Plugin: type ready!
2025-04-22 20:31:48,948 [INFO] root: PluginExample initialized!
2025-04-22 20:31:48,948 [INFO] commands: Plugin: type ready!
2025-04-22 20:31:48,949 [INFO] discord.client: logging in using static token
2025-04-22 20:31:49,948 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 482ae087b25458f076be5e624c028a31).
2025-04-22 20:32:12,020 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:32:12,024 [INFO] commands: Loading 3 plugins
2025-04-22 20:32:12,024 [INFO] root: PluginNewChar initialized!
2025-04-22 20:32:12,024 [INFO] commands: Plugin: type ready!
2025-04-22 20:32:12,025 [INFO] root: PluginMagic initialized!
2025-04-22 20:32:12,025 [INFO] commands: Plugin: type ready!
2025-04-22 20:32:12,025 [INFO] root: PluginExample initialized!
2025-04-22 20:32:12,025 [INFO] commands: Plugin: type ready!
2025-04-22 20:32:12,025 [INFO] discord.client: logging in using static token
2025-04-22 20:32:13,123 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 7f7d6d4d8de3c8dc6dc644a6e94033c8).
2025-04-22 20:32:40,819 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:32:40,828 [INFO] commands: Loading 3 plugins
2025-04-22 20:32:40,828 [INFO] root: PluginNewChar initialized!
2025-04-22 20:32:40,828 [INFO] commands: Plugin: type ready!
2025-04-22 20:32:40,828 [INFO] root: PluginMagic initialized!
2025-04-22 20:32:40,828 [INFO] commands: Plugin: type ready!
2025-04-22 20:32:40,829 [INFO] root: PluginExample initialized!
2025-04-22 20:32:40,829 [INFO] commands: Plugin: type ready!
2025-04-22 20:32:40,829 [INFO] discord.client: logging in using static token
2025-04-22 20:32:41,782 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: acbc7589d7952864978bd4478fa530c0).
2025-04-22 20:33:15,589 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:33:15,597 [INFO] commands: Loading 3 plugins
2025-04-22 20:33:15,597 [INFO] root: PluginNewChar initialized!
2025-04-22 20:33:15,597 [INFO] commands: Plugin: type ready!
2025-04-22 20:33:15,598 [INFO] root: PluginMagic initialized!
2025-04-22 20:33:15,598 [INFO] commands: Plugin: type ready!
2025-04-22 20:33:15,598 [INFO] root: PluginExample initialized!
2025-04-22 20:33:15,598 [INFO] commands: Plugin: type ready!
2025-04-22 20:33:15,599 [INFO] discord.client: logging in using static token
2025-04-22 20:33:16,735 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 3550df50d8a7c1815f878a24a424a862).
2025-04-22 20:33:50,925 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:33:50,932 [INFO] commands: Loading 3 plugins
2025-04-22 20:33:50,933 [INFO] root: PluginNewChar initialized!
2025-04-22 20:33:50,933 [INFO] commands: Plugin: type ready!
2025-04-22 20:33:50,933 [INFO] root: PluginMagic initialized!
2025-04-22 20:33:50,933 [INFO] commands: Plugin: type ready!
2025-04-22 20:33:50,933 [INFO] root: PluginExample initialized!
2025-04-22 20:33:50,933 [INFO] commands: Plugin: type ready!
2025-04-22 20:33:50,934 [INFO] discord.client: logging in using static token
2025-04-22 20:33:51,848 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 0864fdeb27952318e74ea3e44704c889).
2025-04-22 20:34:33,257 [WARNING] discord.client: PyNaCl is not installed, voice will NOT be supported
2025-04-22 20:34:33,266 [INFO] commands: Loading 3 plugins
2025-04-22 20:34:33,266 [INFO] root: PluginNewChar initialized!
2025-04-22 20:34:33,266 [INFO] commands: Plugin: type ready!
2025-04-22 20:34:33,267 [INFO] root: PluginMagic initialized!
2025-04-22 20:34:33,267 [INFO] commands: Plugin: type ready!
2025-04-22 20:34:33,267 [INFO] root: PluginExample initialized!
2025-04-22 20:34:33,267 [INFO] commands: Plugin: type ready!
2025-04-22 20:34:33,268 [INFO] discord.client: logging in using static token
2025-04-22 20:34:34,259 [INFO] discord.gateway: Shard ID None has connected to Gateway (Session ID: 3be7ac73279377d1d6c60154ef904413).
